components/uniset/src/props/mod.rs

```diff
@@ -0,0 +1,1802 @@
+        }
+        CanonicalCombiningClass::CCC13 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC13_V1)
+        }
+        CanonicalCombiningClass::CCC130 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC130_V1)
+        }
+        CanonicalCombiningClass::CCC132 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC132_V1)
+        }
+        CanonicalCombiningClass::CCC133 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC133_V1)
+        }
+        CanonicalCombiningClass::CCC14 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC14_V1)
+        }
+        CanonicalCombiningClass::CCC15 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC15_V1)
+        }
+        CanonicalCombiningClass::CCC16 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC16_V1)
+        }
+        CanonicalCombiningClass::CCC17 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC17_V1)
+        }
+        CanonicalCombiningClass::CCC18 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC18_V1)
+        }
+        CanonicalCombiningClass::CCC19 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC19_V1)
+        }
+        CanonicalCombiningClass::CCC20 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC20_V1)
+        }
+        CanonicalCombiningClass::AttachedBelowLeft => get_prop(
+            provider,
+            key::CANONICAL_COMBINING_CLASS_ATTACHED_BELOW_LEFT_V1,
+        ),
+        CanonicalCombiningClass::AttachedBelow => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_ATTACHED_BELOW_V1)
+        }
+        CanonicalCombiningClass::CCC21 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC21_V1)
+        }
+        CanonicalCombiningClass::AttachedAbove => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_ATTACHED_ABOVE_V1)
+        }
+        CanonicalCombiningClass::AttachedAboveRight => get_prop(
+            provider,
+            key::CANONICAL_COMBINING_CLASS_ATTACHED_ABOVE_RIGHT_V1,
+        ),
+        CanonicalCombiningClass::BelowLeft => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_BELOW_LEFT_V1)
+        }
+        CanonicalCombiningClass::CCC22 => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_CCC22_V1)
+        }
+        CanonicalCombiningClass::Below => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_BELOW_V1)
+        }
+        CanonicalCombiningClass::BelowRight => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_BELOW_RIGHT_V1)
+        }
+        CanonicalCombiningClass::Left => get_prop(provider, key::CANONICAL_COMBINING_CLASS_LEFT_V1),
+        CanonicalCombiningClass::Right => {
+            get_prop(provider, key::CANONICAL_COMBINING_CLASS_RIGHT_V1)
+        }
```

Comment:
How come there's still inconsistent formatting despite the new line length config file?

Reply:
Done -- I'm not sure why `rustfmt.toml` seemed to get ignored after refactoring `props.rs` -> `props/mod.rs` and adding the .toml config into the new directory. But it does work as expected when I keep the .toml config file in the original directory, which is the parent of that `props` dir (`components/uniset/src`). So I reverted the work to create the `props` dir.